Spigot-Server-Patches/0083-Allow-Reloading-of-Custom-Permissions.patch

```diff
@@ -19,9 +19,18 @@ index 5b91630..ab18e1a 100644
 +    public void reloadPermissions() {
 +        ((SimplePluginManager) pluginManager).clearPermissions();
 +        loadCustomPermissions();
++        for (Plugin plugin : pluginManager.getPlugins()) {
++            plugin.getDescription().getPermissions().forEach((perm) -> {
++                try {
++                    pluginManager.addPermission(perm);
++                } catch (IllegalArgumentException ex) {
++                    getLogger().log(Level.WARNING, "Plugin " + plugin.getDescription().getFullName() + " tried to register permission '" + perm.getName() + "' but it's already registered", ex);
++                }
++            });
```

Comment:
It isn't required but it will spam someone's console with a stacktrace rather than a "pretty" log message (for each permission that fails).

Probably best to leave it in, which makes me sad because the alternative is nicer code but /shrug/